[2022-03-14 14:44:50,318] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: simple_dag.get_recipes scheduled__2022-03-14T14:44:00+00:00 [queued]>
[2022-03-14 14:44:50,326] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: simple_dag.get_recipes scheduled__2022-03-14T14:44:00+00:00 [queued]>
[2022-03-14 14:44:50,326] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-03-14 14:44:50,326] {taskinstance.py:1244} INFO - Starting attempt 1 of 1
[2022-03-14 14:44:50,326] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-03-14 14:44:50,336] {taskinstance.py:1264} INFO - Executing <Task(PythonOperator): get_recipes> on 2022-03-14 14:44:00+00:00
[2022-03-14 14:44:50,340] {standard_task_runner.py:52} INFO - Started process 88180 to run task
[2022-03-14 14:44:50,346] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'simple_dag', 'get_recipes', 'scheduled__2022-03-14T14:44:00+00:00', '--job-id', '186', '--raw', '--subdir', 'DAGS_FOLDER/simple_dag.py', '--cfg-path', '/var/folders/tl/ljh9dpdd2fnbgmc62qc2b49h0000gn/T/tmpmse5uxij', '--error-file', '/var/folders/tl/ljh9dpdd2fnbgmc62qc2b49h0000gn/T/tmpodblldcg']
[2022-03-14 14:44:50,349] {standard_task_runner.py:77} INFO - Job 186: Subtask get_recipes
[2022-03-14 14:44:50,406] {logging_mixin.py:109} INFO - Running <TaskInstance: simple_dag.get_recipes scheduled__2022-03-14T14:44:00+00:00 [running]> on host MacBook-Pro.local
[2022-03-14 14:44:50,454] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=simple_dag
AIRFLOW_CTX_TASK_ID=get_recipes
AIRFLOW_CTX_EXECUTION_DATE=2022-03-14T14:44:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-03-14T14:44:00+00:00
[2022-03-14 14:44:50,461] {base_aws.py:190} INFO - Credentials retrieved from login
[2022-03-14 14:44:50,462] {base_aws.py:90} INFO - Retrieving region_name from Connection.extra_config['region_name']
[2022-03-14 14:44:50,843] {base_aws.py:190} INFO - Credentials retrieved from login
[2022-03-14 14:44:50,844] {base_aws.py:90} INFO - Retrieving region_name from Connection.extra_config['region_name']
[2022-03-14 14:44:51,253] {taskinstance.py:1718} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/opt/anaconda3/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 1334, in _run_raw_task
    self._execute_task_with_callbacks(context)
  File "/opt/anaconda3/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 1460, in _execute_task_with_callbacks
    result = self._execute_task(context, self.task)
  File "/opt/anaconda3/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 1516, in _execute_task
    result = execute_callable(context=context)
  File "/opt/anaconda3/lib/python3.9/site-packages/airflow/operators/python.py", line 174, in execute
    return_value = self.execute_callable()
  File "/opt/anaconda3/lib/python3.9/site-packages/airflow/operators/python.py", line 188, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/Users/HomeFolder/airflow/dags/simple_dag.py", line 59, in get_recipes
    data = json.loads(object)
  File "/opt/anaconda3/lib/python3.9/json/__init__.py", line 339, in loads
    raise TypeError(f'the JSON object must be str, bytes or bytearray, '
TypeError: the JSON object must be str, bytes or bytearray, not s3.Object
[2022-03-14 14:44:51,265] {taskinstance.py:1272} INFO - Marking task as FAILED. dag_id=simple_dag, task_id=get_recipes, execution_date=20220314T144400, start_date=20220314T194450, end_date=20220314T194451
[2022-03-14 14:44:51,278] {standard_task_runner.py:89} ERROR - Failed to execute job 186 for task get_recipes
Traceback (most recent call last):
  File "/opt/anaconda3/lib/python3.9/site-packages/airflow/task/task_runner/standard_task_runner.py", line 85, in _start_by_fork
    args.func(args, dag=self.dag)
  File "/opt/anaconda3/lib/python3.9/site-packages/airflow/cli/cli_parser.py", line 48, in command
    return func(*args, **kwargs)
  File "/opt/anaconda3/lib/python3.9/site-packages/airflow/utils/cli.py", line 92, in wrapper
    return f(*args, **kwargs)
  File "/opt/anaconda3/lib/python3.9/site-packages/airflow/cli/commands/task_command.py", line 298, in task_run
    _run_task_by_selected_method(args, dag, ti)
  File "/opt/anaconda3/lib/python3.9/site-packages/airflow/cli/commands/task_command.py", line 107, in _run_task_by_selected_method
    _run_raw_task(args, ti)
  File "/opt/anaconda3/lib/python3.9/site-packages/airflow/cli/commands/task_command.py", line 180, in _run_raw_task
    ti._run_raw_task(
  File "/opt/anaconda3/lib/python3.9/site-packages/airflow/utils/session.py", line 70, in wrapper
    return func(*args, session=session, **kwargs)
  File "/opt/anaconda3/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 1334, in _run_raw_task
    self._execute_task_with_callbacks(context)
  File "/opt/anaconda3/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 1460, in _execute_task_with_callbacks
    result = self._execute_task(context, self.task)
  File "/opt/anaconda3/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 1516, in _execute_task
    result = execute_callable(context=context)
  File "/opt/anaconda3/lib/python3.9/site-packages/airflow/operators/python.py", line 174, in execute
    return_value = self.execute_callable()
  File "/opt/anaconda3/lib/python3.9/site-packages/airflow/operators/python.py", line 188, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/Users/HomeFolder/airflow/dags/simple_dag.py", line 59, in get_recipes
    data = json.loads(object)
  File "/opt/anaconda3/lib/python3.9/json/__init__.py", line 339, in loads
    raise TypeError(f'the JSON object must be str, bytes or bytearray, '
TypeError: the JSON object must be str, bytes or bytearray, not s3.Object
[2022-03-14 14:44:51,304] {local_task_job.py:154} INFO - Task exited with return code 1
[2022-03-14 14:44:51,325] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-03-14 14:47:19,676] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: simple_dag.get_recipes scheduled__2022-03-14T14:44:00+00:00 [queued]>
[2022-03-14 14:47:19,683] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: simple_dag.get_recipes scheduled__2022-03-14T14:44:00+00:00 [queued]>
[2022-03-14 14:47:19,683] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-03-14 14:47:19,683] {taskinstance.py:1244} INFO - Starting attempt 1 of 1
[2022-03-14 14:47:19,684] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-03-14 14:47:19,694] {taskinstance.py:1264} INFO - Executing <Task(PythonOperator): get_recipes> on 2022-03-14 14:44:00+00:00
[2022-03-14 14:47:19,698] {standard_task_runner.py:52} INFO - Started process 88286 to run task
[2022-03-14 14:47:19,706] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'simple_dag', 'get_recipes', 'scheduled__2022-03-14T14:44:00+00:00', '--job-id', '196', '--raw', '--subdir', 'DAGS_FOLDER/simple_dag.py', '--cfg-path', '/var/folders/tl/ljh9dpdd2fnbgmc62qc2b49h0000gn/T/tmphwdekz4q', '--error-file', '/var/folders/tl/ljh9dpdd2fnbgmc62qc2b49h0000gn/T/tmp_ttox7ru']
[2022-03-14 14:47:19,709] {standard_task_runner.py:77} INFO - Job 196: Subtask get_recipes
[2022-03-14 14:47:19,773] {logging_mixin.py:109} INFO - Running <TaskInstance: simple_dag.get_recipes scheduled__2022-03-14T14:44:00+00:00 [running]> on host 1.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.ip6.arpa
[2022-03-14 14:47:19,826] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=simple_dag
AIRFLOW_CTX_TASK_ID=get_recipes
AIRFLOW_CTX_EXECUTION_DATE=2022-03-14T14:44:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-03-14T14:44:00+00:00
[2022-03-14 14:47:19,835] {base_aws.py:190} INFO - Credentials retrieved from login
[2022-03-14 14:47:19,836] {base_aws.py:90} INFO - Retrieving region_name from Connection.extra_config['region_name']
[2022-03-14 14:47:20,277] {base_aws.py:190} INFO - Credentials retrieved from login
[2022-03-14 14:47:20,281] {base_aws.py:90} INFO - Retrieving region_name from Connection.extra_config['region_name']
[2022-03-14 14:47:20,797] {logging_mixin.py:109} INFO - s3.Object(bucket_name='airflowrs', key='raw/user-2022-03-14 12:16:16.768057.json')
[2022-03-14 14:47:20,803] {base_aws.py:190} INFO - Credentials retrieved from login
[2022-03-14 14:47:20,804] {base_aws.py:90} INFO - Retrieving region_name from Connection.extra_config['region_name']
[2022-03-14 14:47:21,125] {logging_mixin.py:109} INFO - s3.Object(bucket_name='airflowrs', key='raw/user-2022-03-14 12:16:24.788633.json')
[2022-03-14 14:47:21,130] {base_aws.py:190} INFO - Credentials retrieved from login
[2022-03-14 14:47:21,131] {base_aws.py:90} INFO - Retrieving region_name from Connection.extra_config['region_name']
[2022-03-14 14:47:21,459] {logging_mixin.py:109} INFO - s3.Object(bucket_name='airflowrs', key='raw/user-2022-03-14 12:16:29.962475.json')
[2022-03-14 14:47:21,465] {base_aws.py:190} INFO - Credentials retrieved from login
[2022-03-14 14:47:21,466] {base_aws.py:90} INFO - Retrieving region_name from Connection.extra_config['region_name']
[2022-03-14 14:47:21,777] {logging_mixin.py:109} INFO - s3.Object(bucket_name='airflowrs', key='raw/user-2022-03-14 12:16:44.008253.json')
[2022-03-14 14:47:21,781] {base_aws.py:190} INFO - Credentials retrieved from login
[2022-03-14 14:47:21,781] {base_aws.py:90} INFO - Retrieving region_name from Connection.extra_config['region_name']
[2022-03-14 14:47:22,104] {logging_mixin.py:109} INFO - s3.Object(bucket_name='airflowrs', key='raw/user-2022-03-14 12:16:52.065604.json')
[2022-03-14 14:47:22,109] {base_aws.py:190} INFO - Credentials retrieved from login
[2022-03-14 14:47:22,110] {base_aws.py:90} INFO - Retrieving region_name from Connection.extra_config['region_name']
[2022-03-14 14:47:22,535] {logging_mixin.py:109} INFO - s3.Object(bucket_name='airflowrs', key='raw/user-2022-03-14 13:40:51.952380.json')
[2022-03-14 14:47:22,540] {base_aws.py:190} INFO - Credentials retrieved from login
[2022-03-14 14:47:22,541] {base_aws.py:90} INFO - Retrieving region_name from Connection.extra_config['region_name']
[2022-03-14 14:47:22,869] {logging_mixin.py:109} INFO - s3.Object(bucket_name='airflowrs', key='raw/user-2022-03-14 13:40:54.048138.json')
[2022-03-14 14:47:22,873] {base_aws.py:190} INFO - Credentials retrieved from login
[2022-03-14 14:47:22,875] {base_aws.py:90} INFO - Retrieving region_name from Connection.extra_config['region_name']
[2022-03-14 14:47:23,214] {logging_mixin.py:109} INFO - s3.Object(bucket_name='airflowrs', key='raw/user-2022-03-14 13:40:56.140666.json')
[2022-03-14 14:47:23,219] {base_aws.py:190} INFO - Credentials retrieved from login
[2022-03-14 14:47:23,221] {base_aws.py:90} INFO - Retrieving region_name from Connection.extra_config['region_name']
[2022-03-14 14:47:23,559] {logging_mixin.py:109} INFO - s3.Object(bucket_name='airflowrs', key='raw/user-2022-03-14 13:41:02.424720.json')
[2022-03-14 14:47:23,565] {base_aws.py:190} INFO - Credentials retrieved from login
[2022-03-14 14:47:23,566] {base_aws.py:90} INFO - Retrieving region_name from Connection.extra_config['region_name']
[2022-03-14 14:47:23,880] {logging_mixin.py:109} INFO - s3.Object(bucket_name='airflowrs', key='raw/user-2022-03-14 13:41:04.521027.json')
[2022-03-14 14:47:23,884] {base_aws.py:190} INFO - Credentials retrieved from login
[2022-03-14 14:47:23,886] {base_aws.py:90} INFO - Retrieving region_name from Connection.extra_config['region_name']
[2022-03-14 14:47:24,325] {logging_mixin.py:109} INFO - s3.Object(bucket_name='airflowrs', key='raw/user-2022-03-14 13:41:06.619964.json')
[2022-03-14 14:47:24,330] {base_aws.py:190} INFO - Credentials retrieved from login
[2022-03-14 14:47:24,331] {base_aws.py:90} INFO - Retrieving region_name from Connection.extra_config['region_name']
[2022-03-14 14:47:24,858] {logging_mixin.py:109} INFO - s3.Object(bucket_name='airflowrs', key='raw/user-2022-03-14 13:41:08.717143.json')
[2022-03-14 14:47:24,863] {base_aws.py:190} INFO - Credentials retrieved from login
[2022-03-14 14:47:24,864] {base_aws.py:90} INFO - Retrieving region_name from Connection.extra_config['region_name']
[2022-03-14 14:47:25,246] {logging_mixin.py:109} INFO - s3.Object(bucket_name='airflowrs', key='raw/user-2022-03-14 13:41:10.809943.json')
[2022-03-14 14:47:25,251] {base_aws.py:190} INFO - Credentials retrieved from login
[2022-03-14 14:47:25,253] {base_aws.py:90} INFO - Retrieving region_name from Connection.extra_config['region_name']
[2022-03-14 14:47:25,564] {logging_mixin.py:109} INFO - s3.Object(bucket_name='airflowrs', key='raw/user-2022-03-14 13:41:12.902259.json')
[2022-03-14 14:47:25,570] {base_aws.py:190} INFO - Credentials retrieved from login
[2022-03-14 14:47:25,571] {base_aws.py:90} INFO - Retrieving region_name from Connection.extra_config['region_name']
[2022-03-14 14:47:26,295] {logging_mixin.py:109} INFO - s3.Object(bucket_name='airflowrs', key='raw/user-2022-03-14 13:41:14.996433.json')
[2022-03-14 14:47:26,301] {base_aws.py:190} INFO - Credentials retrieved from login
[2022-03-14 14:47:26,302] {base_aws.py:90} INFO - Retrieving region_name from Connection.extra_config['region_name']
[2022-03-14 14:47:26,633] {logging_mixin.py:109} INFO - s3.Object(bucket_name='airflowrs', key='raw/user-2022-03-14 13:41:19.188289.json')
[2022-03-14 14:47:26,637] {base_aws.py:190} INFO - Credentials retrieved from login
[2022-03-14 14:47:26,638] {base_aws.py:90} INFO - Retrieving region_name from Connection.extra_config['region_name']
[2022-03-14 14:47:27,080] {logging_mixin.py:109} INFO - s3.Object(bucket_name='airflowrs', key='raw/user-2022-03-14 13:41:21.281386.json')
[2022-03-14 14:47:27,084] {base_aws.py:190} INFO - Credentials retrieved from login
[2022-03-14 14:47:27,085] {base_aws.py:90} INFO - Retrieving region_name from Connection.extra_config['region_name']
[2022-03-14 14:47:27,577] {logging_mixin.py:109} INFO - s3.Object(bucket_name='airflowrs', key='raw/user-2022-03-14 13:41:23.377181.json')
[2022-03-14 14:47:27,581] {base_aws.py:190} INFO - Credentials retrieved from login
[2022-03-14 14:47:27,583] {base_aws.py:90} INFO - Retrieving region_name from Connection.extra_config['region_name']
[2022-03-14 14:47:27,921] {logging_mixin.py:109} INFO - s3.Object(bucket_name='airflowrs', key='raw/user-2022-03-14 13:41:25.468979.json')
[2022-03-14 14:47:27,925] {base_aws.py:190} INFO - Credentials retrieved from login
[2022-03-14 14:47:27,926] {base_aws.py:90} INFO - Retrieving region_name from Connection.extra_config['region_name']
[2022-03-14 14:47:28,254] {logging_mixin.py:109} INFO - s3.Object(bucket_name='airflowrs', key='raw/user-2022-03-14 13:41:27.559613.json')
[2022-03-14 14:47:28,261] {python.py:175} INFO - Done. Returned value was: None
[2022-03-14 14:47:28,270] {taskinstance.py:1272} INFO - Marking task as SUCCESS. dag_id=simple_dag, task_id=get_recipes, execution_date=20220314T144400, start_date=20220314T194719, end_date=20220314T194728
[2022-03-14 14:47:28,318] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-03-14 14:47:28,338] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
